cli/lib/kontena/client.rb

```diff
@@ -46,7 +46,7 @@ def initialize(api_url, token = nil, options = {})
       uri = URI.parse(@api_url)
       @host = uri.host
 
-      @logger = Logger.new(STDOUT)
+      @logger = Logger.new(ENV["DEBUG"] ? STDERR : STDOUT)
```

Comment:
At least here this can just be `STDERR`, since there are only `logger.debug ...` calls in the class?